app/controllers/miq_ae_class_controller.rb

```diff
@@ -2752,4 +2755,22 @@ def process_element_destroy_via_queue(element, klass, name)
                {:model => model_name, :name => record_name, :error_msg => bang.message}, :error)
     end
   end
+
+  def breadcrumbs_options
+    @breadcrumbs_start = [{:title => _("Automation")}, {:title => _("Automate")}, {:title => _("Explorer")}]
+  end
+
+  def accord_name
```

Comment:
These seem to already exist in BreadcrumbMixin

Reply:
There is a difference, this controller uses a different naming in `features`:

miq_ae_class
`return feature.title if feature.accord_name == x_active_accord.to_s`

mixin
`return feature.title if feature.name == x_active_accord`

Reply:
And is the content actually different? or does it in the end do the same?

In the later case, please, unify.